content/html/concepts/attributes/terms/title/title.md

```diff
@@ -0,0 +1,33 @@
+**Title**
+
+**HTML global attribute**
+
+In the browser the ‘title’ attribute displays additional information about the HTML element in the form of a mouse over tooltip.
+
+**Syntax**
```

Comment:
Do not use bold for headers, use the Markdown header syntax. You can learn about Markdown in [Codecademy's Markdown Style Guide](http://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/).

Typically we want h2 headers in the body of an entry. In this case it would appear as `## Syntax` . (Two `#` at the start of the line for an h2 header.)